cypress/e2e/pages/photoGalleryPage/tests.js

```diff
@@ -7,12 +7,23 @@ export const testsThatFollowSmokeTestConfig = ({
 }) => {
   describe(`testsThatFollowSmokeTestConfig to run for ${service} ${variant} ${pageType}`, () => {
     it('should render a description for the page', () => {
-      cy.getPageData({ service, pageType: 'cpsAsset', variant }).then(
-        ({ body }) => {
-          const description = body.data.article.promo.summary;
-          cy.get('main p').first().should('contain', description);
-        },
-      );
+      if (['thai', 'pidgin', 'hausa'].includes(service)) {
```

Comment:
Should we just consider making this an article integration test? I don't see the value of this E2E if it's just checking that there's a description on the page...? 

We could save the json for a PGL for one of these services, and perhaps add a new test to get the contents of `main p`

Or is that scope creep for this work? Maybe this is something that we could address later, but I wondered if we could just take the opportunity whilst the CAF work is in progress

Reply:
(Oops I had forgotten that I already made that comment above: https://github.com/bbc/simorgh/pull/11502#issuecomment-2047011721 🤦‍♀️ I still stand by it, but happy to approve so that you can progress, and we will look at this in future with Meriam & Silpa)

Reply:
Yeah sorry, yeah I agree, it's just we need to get this over the line, this is just to get the CAF pages released, and then I think we can include the porting over to integration tests in a later ticket